app.rb

```diff
@@ -31,6 +32,10 @@ def dashboard_url
   def public_url
      ENV['OOD_PUBLIC_URL'] || "/public"
   end
+  
+  def alert(message)
+      "<div class='row'><div class='alert alert-warning' role='alert'>#{message}</div></div>"
+  end

```

Comment:
Should this be an erb instead of a helper? Not sure. Regardless, `message` needs to be HTML escaped.